assets/js/components/settings/settings-modules.js

```diff
@@ -91,11 +92,16 @@ class SettingsModules extends Component {
 				} );
 				return;
 			}
+
 			this.setState( { isSaving: module } );
 			if ( ! modulePromise ) {
 				return;
```

Comment:
@felixarntz suggested it should be called before returning in this block as well. `if ( ! modulePromise )`

Reply:
@aaemnnosttv fixed in https://github.com/google/site-kit-wp/pull/162/commits/e7a276305689c9ce64f067d35e2dfb9effea4f75